.env

```diff
@@ -0,0 +1,2 @@
+FIREBASE_API_KEY =  "AIzaSyCaOmjYj5QIZrwUdRpks_0Cmjj-kmCNaRw";
+
```

Comment:
To w ogóle ciekawe, bo dodałaś .env do gitignore :D Zakładam, że to było tak:
dodałaś plik, zrobiłaś commita, dodałaś .env do gitignore
Zastanów się tylko czy ma to tu na pewno być :D

Reply:
Zgadza się